server/service/src/requisition/response_requisition/insert_program.rs

```diff
@@ -218,10 +218,10 @@ fn generate(
         .query_one(StoreFilter::new().name_id(EqualFilter::equal_to(&other_party_id)))?;
 
     let indicator_values = match customer_store {
-        Some(_) => generate_program_indicator_values(
+        Some(customer_store) => generate_program_indicator_values(
             &ctx.store_id,
             &period_id,
-            &other_party_id,
+            &customer_store.name_row.id,
```

Comment:
For response requisitions, the other_party_id should be the customer's name.id, no?

Reply:
You're right! Unnecessary change. Will revert